MacOSX/build-package.in

```diff
@@ -74,8 +74,8 @@ xcodebuild -configuration Deployment -project OpenSC.tokend/Tokend.xcodeproj
 
 # Prepare target root
 # Copy Tokend
-mkdir -p target/System/Library/Security/tokend
-mv OpenSC.tokend/build/OpenSC.tokend target/System/Library/Security/tokend
+#mkdir -p target/System/Library/Security/tokend
+#mv OpenSC.tokend/build/OpenSC.tokend target/System/Library/Security/tokend
```

Comment:
This change was introduced by accident with 3c5e855261ad88cc537f02dc8fd8c4282b8d3f26 and should be removed